src/index.js

```diff
@@ -1 +1,37 @@
 import './style.css';
+import './images/tv-logo.jpg';
+
+const baseURL = 'https://api.tvmaze.com/search/shows?q=';
+const query = 'star';
+const URL = baseURL + query;
```

Comment:
#24 
**Hi Hans,** 

Good job, you are almost there.
![awesome](https://user-images.githubusercontent.com/46789910/176259991-68356355-57f0-4533-b7e5-09cf58a59086.gif)

Based on the baseURL it will be nice to have a form or category where users can select the category in your case query to change the movies that are being displayed.

**We can also outsource this part into another module and call the module from an arrow function.**

Reply:
Agree!!!